chapter_04/p04_check_balanced.py

```diff
@@ -127,6 +127,37 @@ def test_is_balanced():
             error_msg = f"{is_balanced.__name__} failed on {tree_gen.__name__}"
             assert is_balanced(tree_gen()) == expected, error_msg
 
+#Alternative Recursive Approach
+
+# Balanced Tree
+class Node():
+    def __init__(self, value, left=None, right=None):
+        self.value = value
+        self.left = left
+        self.right = right
+
+
+def helper(root):
```

Comment:
This is too generic of a function name. Names should describe what the function does. In this case, `_find_height` would be a good name.